apps/meteor/app/authorization/server/functions/upsertPermissions.ts

```diff
@@ -221,7 +221,9 @@ export const upsertPermissions = async (): Promise<void> => {
 		{ _id: 'remove-slackbridge-links', roles: ['admin'] },
 		{ _id: 'view-import-operations', roles: ['admin'] },
 		{ _id: 'clear-oembed-cache', roles: ['admin'] },
-		{ _id: 'videoconf-ring-users', roles: ['admin', 'owner', 'moderator', 'user'] },
+		{ _id: 'videoconf-ring-users', roles: ['admin', 'owner4', 'moderator', 'user'] },
```

Comment:
```suggestion
		{ _id: 'videoconf-ring-users', roles: ['admin', 'owner', 'moderator', 'user'] },
```

Reply:
I really don't know how these '4' slipped into it 🧐